src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -205,8 +209,7 @@ public ServerMock()
 			logger.warning("Could not load build properties");
 		}
 
-		logger.setLevel(Level.ALL);
-
+		loadDefaultIcon();
```

Comment:
Is that something we want to cache?

Reply:
That’s how the CraftBukkit implementation works. Perhaps we could add a setter? Not really sure what a use case of that would be. Getting it from disk each call would be deaviating too much since you can’t add the file later and have it update on a real server.

Reply:
Yeah, my main concern is that the icon is loaded for every call to `MockBukkit.mock()` and most tests don't need the icon. But the icon is small enough, I guess it shouldn't be an issue to do this I/O operation each time.

Reply:
It doesn't seem to have much of an I/O load. Just looking at task manager while running tests, it looks the same, even on an HDD. I'm still not sure if it's necessary though, as putting an icon there before mocking would be a bit weird. Should we just have a setter instead?

Reply:
In my opinion, unit tests should not depend on the file system, it's something for integration tests. In MockBukkit, a world is only in memory. On the another hand, tags are stored as json files and are not lazily initialized, they are read every time.

Having the capacity to just set the icon (`setServerIcon(CachedServerIconMock)`?) would be great for swapping things only in memory.

Reply:
Yup, I fully agree. I've removed the automatic loading and added a setter.